Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -3,7 +3,10 @@ Copyright (c) 2021 Kyle Miller. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kyle Miller
 -/
+import Mathlib.Algebra.Order.Ring.Nat
 import Mathlib.Combinatorics.SimpleGraph.Subgraph
+import Mathlib.Data.Fintype.Card
+import Mathlib.Order.ConditionallyCompleteLattice.Basic
```

Comment:
That is quite some new imports in an already long file. Maybe put the new lemmas (and any already existing related lemmas) in a new file?

I also don't see how the new imports are actually used in the new lemmas?

Reply:
This was actually a leftover, the imports are only needed for `WalkCounting.lean`, fixed. (I don't know why the treeshaking didn't pick up on this?)

Fixed.